app/helpers/practical_supports_helper.rb

```diff
@@ -31,10 +31,9 @@ def practical_support_source_options(current_value = nil)
 
   def practical_support_guidance_link
     url = Config.find_or_create_by(config_key: 'practical_support_guidance_url').options.first
-
-    content_tag :small, class: 'float-right' do
-      link_to t('patient.practical_support.guidance_link', fund: FUND), url, target: '_blank'
-    end if url.present?
+    link_content = link_to t('patient.practical_support.guidance_link', fund: FUND), url, target: '_blank'
+    link_content if url.present?
```

Comment:
I do really like the simplification here and we have your insight about the `small` tag not really serving us to thank for it.